2024/wg-fedid.html

```diff
@@ -246,7 +246,7 @@ <h3>Tentative Deliverables</h3>
         <dl>
         <dt id="digid" class="spec"><a href="https://wicg.github.io/digital-identities/">Digital Credentials API</a></dt>
         <dd>
-            <p>This specification defines an API that enables user agents to mediate access to and presentation of Digital Credentials in a format-agnostic and protocol-agnostic fashion (e.g., W3C Verifiable Credentials, ISO mDoc, etc.), enabling different use cases such as - but not limited to - government-issued documents, academic credentials, IoT and Supply Chain related identities.</p>
+            <p>This specification defines an API that enables user agents to mediate access to and presentation of Digital Credentials in a format-agnostic (e.g., W3C Verifiable Credentials, ISO mDoc, IETF SD-JWT VC, etc.) and protocol-agnostic fashion (e.g. OpenID4VC, W3C Verifiable Credentials API, etc.), enabling different use cases such as - but not limited to - government-issued documents, academic credentials, IoT and Supply Chain related identities.</p>
```

Comment:
Digital Bazaar objects to the addition of "IETF SD-JWT VC". If IETF SD-JWT VC is to be put into the Charter, the name, purpose, specification, and understanding must be deconflicted with W3C Verifiable Credentials. At present, we have two specifications at W3C and IETF that use the name "Verifiable Credentials" but have very different data models and meanings that has been and will continue to cause confusion in the marketplace. We should not further this confusion by not, at least, deconflicting the names and purposes of the specifications under consideration.

Reply:
@msporny thanks for the comment. To give you context, this is the message regarding this PR.

https://lists.w3.org/Archives/Public/public-review-comments/2024Sep/0006.html

To clarify the difference, do you have any specific suggestions?

Reply:
I wouldn't object to: "credential formats produced by the IETF"

Reply:
updated, then we can try to deconflict in some ways.

Personally when I wrote the [identity report](https://www.w3.org/reports/identity-web-impact/) I spent a lot of time on the terminology part, precisely to try to get clarity on different terms but with the same meaning and the same terms with different meanings according to the context.

The purpose is to get clarity and to collaborate between W3C and IETF for the good of the users, since in the end with the implementation on government credentials we all get hurt as citizens.